clang/include/clang/Basic/DiagnosticGroups.td

```diff
@@ -1257,6 +1257,7 @@ def Sycl2020Compat : DiagGroup<"sycl-2020-compat">;
 def SyclStrict : DiagGroup<"sycl-strict", [ Sycl2017Compat, Sycl2020Compat]>;
 def SyclTarget : DiagGroup<"sycl-target">;
 def SyclFPGAMismatch : DiagGroup<"sycl-fpga-mismatch">;
+def SyclIvdepAttribute : DiagGroup<"ivdep-compat">;
```

Comment:
This should be grouped under `IgnoredAttributes` so that `-Wno-ignored-attributes` also works properly.